uswid/container.py

```diff
@@ -61,6 +61,10 @@ def get_default(self) -> Optional[uSwidIdentity]:
         """returns the existing identity, or creates one if none already exist"""
 
         if len(self._identities) > 1:
+            print("Found multiple tag IDs:", len(self._identities))
```

Comment:
It would probably be nice to have something like:
"Error: multiple tag ID's are not supported"
so the user knows that's the reason it fails.

Reply:
Yeah currently that message is per format, e.g.:

> cannot save JSON when no default identity

So I just wanted to add the list above. WDYT, should it be like "Error: exporting multiple tag ID's is not supported"?

Reply:
I mean per error message for each format

Reply:
I think we just return a JSON array, no?

Reply:
currently python-uswid is erroring when it found multiple blobs:
https://github.com/hughsie/python-uswid/blob/84c879a720d5cb7a2f41acc5a7ddbf1aefebc84c/uswid/cli.py#L334

which is evaluated as "no default identity", coming from here:
https://github.com/hughsie/python-uswid/blob/84c879a720d5cb7a2f41acc5a7ddbf1aefebc84c/uswid/container.py#L63

Reply:
You can also just replace the "cannot sae JSON when no default identity" with the error message. Of course you would need to to this for each format that currently does not support multiple tags. But I guess it would be cleaner this way.